files/scripts/arp_update

```diff
@@ -22,10 +22,10 @@ while /bin/true; do
       fi
       for nexthop in $STATIC_ROUTE_NEXTHOPS; do
           if [[ $nexthop == *"."* ]]; then
-              neigh_state=( $(ip -4 neigh show | grep $nexthop | tr -s ' ' | cut -d ' ' -f 3,4) )
+              neigh_state=( $(ip -4 neigh show | grep -w $nexthop | tr -s ' ' | cut -d ' ' -f 3,4) )
```

Comment:
hi @abdosi , can you please provide in description with an example of what was before and what is return with this options. Will be good to refer in future for such changes

Reply:
@prsunny updated the PR description with the example